nodejs-jwt-example/index.js

```diff
@@ -1,13 +1,12 @@
 const express = require("express");
-const bodyParser = require("body-parser");
 const jwt = require("jsonwebtoken");
 const url = require("url");
 const escapeHtml = require("escape-html");
 
 const PORT = process.env.PORT || 3535;
 const METABASE_JWT_URL = "http://localhost:3000/auth/sso";
 const METABASE_JWT_SHARED_SECRET =
-  "REPLACE WITH KEY FROM http://localhost:3000/admin/settings/authentication/jwt";
+  "56aac251b84fb05dc4a63a254156e9ccd328f284eb415391c0ce35c01bb47d03";
```

Comment:
Can't we load this from the env or something?

Reply:
we could, but I added in the readme how to populate this